proto/tripcommon.proto

```diff
@@ -112,6 +112,17 @@ message TransitStationInfo {
   optional LatLng ll = 3;                // Latitude, longitude of the station
 }
 
+message BikeShareStationInfo {
+  optional string name = 1;
+  optional string ref = 2;
+  optional string capacity = 3;
+  optional string network = 4;
+  optional string operator = 5;
+  optional float rent_cost = 6;
+  optional float return_cost = 7;
+  optional string osm_node_id = 8;
```

Comment:
A couple things about this. This should be and unsigned 64bit value, definitely not a string but also we have another in progress pr that adds support for this which is already quite close to done. Would you be OK to remove this with the knowledge that we will soon have support for osm node ids from this PR #3245 by @sleepingwell?

Reply:
Perfect I'll remove it :)